irodsbackend/static/irodsbackend/js/irodsbackend.js

```diff
@@ -254,11 +254,18 @@ $(document).ready(function() {
                         colSpan = '4';
                         icon = 'mdi:folder-open';
                         toolTip = 'Collection';
+
+                        var copyButton = $('<button>')
```

Comment:
The buttons are not currently pulled to the right side of the table column as specified in the spec. Instead, they are directly behind the collection or file title. If `pull-right` doesn't work, maybe `ml-auto` does. I forget which but you can find examples from existing tables and experiment.